koa-crud/src/graphql/typeDefs.ts

```diff
@@ -0,0 +1,36 @@
+import { gql } from 'apollo-server-koa';
+
+export default gql`
+  type Query {
+    vendors: ArrayOfVendorReturn
```

Comment:
Simplify naming, as much as possible we should remove technical jargons in our variable name.
```
Plural = Vendors
Singular = Vendor
```

Reply:
noted sir will update on my next PR